dvc/command/update.py

```diff
@@ -23,7 +23,7 @@ def run(self):
 
 
 def add_parser(subparsers, parent_parser):
-    UPDATE_HELP = "Update dependencies and reproduce specified DVC-files."
+    UPDATE_HELP = "Update data artifacts imported from other DVC repositories."
```

Comment:
Please note this change is due to https://github.com/iterative/dvc.org/pull/490#discussion_r305584059. Feel free to resolve this comment after reading.